PageObjects/APII/LoginAPI.js

```diff
@@ -0,0 +1,69 @@
+import endpoints from "../../Fixtures/endpoints.json";
+import dotenv from "dotenv";
+import path from "path";
+import { expect } from "@playwright/test";
+dotenv.config({ path: path.resolve(__dirname, "../.env") });
+
```

Comment:
if you look at swager what do you have? auth group right? and there you have methods login, register etc... same should be here. Class should be Auth and methods should be login, register etc